packages/editing-adapter/src/util/create_object_url.ts

```diff
@@ -35,7 +35,11 @@ export function createSafeObjectURL(blob: Blob): SafeObjectUrl {
 
 /** Returns true if the given raw URL is an object URL. */
 export function isObjectUrl(url: string): boolean {
-  return new URL(url).protocol === 'blob:';
+  try {
+    return new URL(url).protocol === 'blob:';
+  } catch (_) {
+    return false;
+  }
```

Comment:
It looks like you branched off your other PR instead of off master.

Reply:
pulled and merged.